tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -232,22 +236,45 @@ def create_statistics_html(self, output_dir):
                 checker = report['checkerName']
                 checker_statistics[checker] += 1
 
-        rows = []
+        checker_rows = []
         checker_statistics_content = ''
+        severity_statistics = defaultdict(int)
         for checker_name in sorted(checker_statistics):
             checker_statistics_content += '''
               <tr>
                 <td>{0}</td>
+                <td class="severity" severity="{1}">
+                  <i class="severity-{1}"></i>
```

Comment:
It would be good to add a `title` attribute to this element or for the td element so when the user hover the mouse over this column he will see what does the C severity means (critical).